packages/site/cms-site/src/blocks/PixelImageBlock.tsx

```diff
@@ -11,14 +12,10 @@ import { PropsWithData } from "./PropsWithData";
 
 interface PixelImageBlockProps extends PropsWithData<PixelImageBlockData>, Omit<ImageProps, "src" | "width" | "height" | "alt"> {
     aspectRatio: string | "inherit";
```

Comment:
@johnnyomair @mariokemetinger should we change this "inherit" string to something else? (eg. "file") 
If this is set, the aspectRatio from the file is calculated. 
Inherit would mean, that the dimensions from the parent should be used.
The function can be called `computeAspectRatioFromCrop`